Vagrantfile

```diff
@@ -199,13 +209,23 @@ Vagrant.configure("2") do |config|
     node_name = "node#{i}"
 
     config.vm.define node_name, autostart: false do |node|
+      if i > 5
```

Comment:
A simple suggestion to make things more obvious for users (as @slaperche-scality mentioned, using a node's number to choose which OS to install is kind of hackish):

Maybe you could instead duplicate the block defining `node#{i}` VMs and rename them `centos#{i}` and `ubuntu#{i}`. I'm also checking around Vagrant docs to see if we could parametrize this more easily (cc @nootal, any idea?).

Reply:
Let's go for this simple solution right now, maybe we'll want to handle user configuration (through a file) in a later iteration.

Reply:
Yes, please use the OS name in the node name, and always count from 1.

Reply:
Change done.